sdk/core/azure-core-http-netty/CHANGELOG.md

```diff
@@ -3,6 +3,9 @@
 ## 1.12.0-beta.1 (Unreleased)
 
 ### Features Added
+- The `NettyAsyncHttpClientProvider.createInstance()` now have option to share one single shared `HttpClient`.
+  A new configuration environment variable `AZURE_ENABLE_HTTP_CLIENT_SHARING` has been added for backwards
+  compatibility. Setting it to `true` before starting the process to use single shared `HttpClient`.
```

Comment:
Since the flag has changed from opt-out to opt-in this statement can be changed.

```suggestion
  Set the environment variable `AZURE_ENABLE_HTTP_CLIENT_SHARING`  to `true` before starting the process to use the shared `HttpClient`.
```